playwright-e2e/pages/base-page.ts

```diff
@@ -24,18 +22,25 @@ export class BasePage {
     await this.go.click();
   }
 
-  async checkYourAnsAndSubmit() {
-    await this.saveAndContinue.click();
+  async checkYourAnsAndSubmit(submitLabel: string = "Save and continue") {
```

Comment:
is there a reason to change save and continue?

Reply:
It has a default so shouldn’t impact any other test classes